bin-or-misc/theme-nextjs-integration.tsx

```
@@ -0,0 +1,369 @@
+ *   color: var(--color-foreground);
+ *   transition: background-color 0.3s ease, color 0.3s ease;
+ * }
+ */
+
+// ============================================================================
+// 4. Theme Toggle Component (components/ThemeToggle.tsx)
+// ============================================================================
+
+'use client';
+
+import { useTheme } from '@/app/providers';
```

Biome (2.1.2)

[error] Shouldn't redeclare 'useTheme'. Consider to delete it or rename it.

'useTheme' is defined here:


(lint/suspicious/noRedeclare)